test/jit/test_builtins.py

```diff
@@ -134,3 +136,34 @@ def fn(x):
             if p in EQUALITY_MISMATCH:
                 continue
             self.assertEqual(getattr(tensor, p), cu.fn(tensor))
+
+    def test_tensor_subscript_assign(self):
+        def fn1(x):
```

Comment:
Can you add some tests on GPU as well ? You'll need to guard those tests as in https://github.com/pytorch/pytorch/blob/master/test/test_jit_cuda_fuser.py#L55